exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting = "I am BOshra";
```

Comment:
Hi there @BoshraM, 
We've just updated our coursework with a change to the variable declarations.
From now on when you're writing variable declarations be sure to use `let` / `const` over `var`.
```suggestion
const greeting = "I am BOshra";
```